### DISCUSSION TITLE
How to tune gamma inside TPE

### DISCUSSION THREAD

**Discussion by VivekSM1992** [optuna/optuna#4115] [Q&A]
Could you please show an example to tune gamma function?. in the website is says that refer the paper. However, I couldn't find how to define gamma function.

**Comment by amylase** [comment]
Could you elaborate your question? What do you exactly want to achieve? I can think of:
* You want to know the mathematical definition of gamma function
* You want to know how to obtain the value of gamma function in Python
  * You can use standard library: https://docs.python.org/3.11/library/math.html#math.gamma
* You want to know how to define the objective function that returns the gamma function value for Optuna.
* and so on...
